libs/R-TypeLogic/Global/Systems/MovementSystem.cpp

```diff
@@ -37,6 +37,8 @@ void Movement::run(World &world)
             if ((pos.y >= 1072 && vel.multiplierOrdinate > 0))
                 return;
         }
+        if (elapsedTimeInSeconds >= 1)
```

Comment:
Why ?

Reply:
Refering to the PR description : "Add an anti-lag functionality inside movement system (When there is more than one second of elapsed time, no movement is applied)"